src/views/Catalog.vue

```diff
@@ -5,25 +5,53 @@
         <ion-title>{{ $t("Catalog") }}</ion-title>
       </ion-toolbar>
     </ion-header>
-   
-    <ion-content :fullscreen="true">
-    
-    </ion-content>
+    <ion-content :fullscreen="true">    
+      <ion-searchbar />
+      <ion-row>
```

Comment:
Please use css grid for this instead of row and column. Take reference of preorder app.
https://github.com/hotwax/preorder

Reply:
Used css grid instead. 
Done